diff --git a/Makefiles/Makefile.UNIX b/Makefiles/Makefile.UNIX
index af02998..9ba87d2 100644
--- a/Makefiles/Makefile.UNIX
+++ b/Makefiles/Makefile.UNIX
@@ -73,7 +73,7 @@ endif
 #
 
 PLATFORM_CXXFLAGS = $(SDL2_CFLAGS) -O3
-PLATFORM_LDFLAGS = $(SDL2_LIBS) -lGL -lGLU -lz -lm -lstdc++ -lpthread
+PLATFORM_LDFLAGS = $(SDL2_LIBS) -lGL -lGLU -lz -lm -lstdc++ -lpthread -L/app/lib
 PLATFORM_BINARY_EXTENSION =
 
 
